src/components/custom-components/CustomText.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  Poppins_400Regular, useFonts
+} from '@expo-google-fonts/poppins';
+import AppLoading from 'expo-app-loading';
+import * as React from 'react';
+import { Text } from 'react-native';
+
+interface CustomTextProps {
+  children: string,
+  style?: {
+    
+  }
+}
+
+
+
+
+const CustomText = (props: CustomTextProps) => {
+    
+  const [fontsLoaded] = useFonts({
+    Poppins_400Regular,
+  });
+    
+  if (!fontsLoaded) {
+    return <AppLoading />;
+  } else { return (
+    <Text {...props}>{props.children}</Text>
+
+  );}
+
+    
+};
+
+
+
+
```

Comment:
Do you have linter?

Reply:
Yes, i have linter.
I'll revision my eslint conig